src/models/TestResultsSchemaHGVCancelled.ts

```diff
@@ -21,8 +21,17 @@ const testTypesSchema = Joi.object().keys({
     ...testTypesCommonSchema,
     testResult: Joi.any().only(["fail", "pass", "prs", "abandoned"]).required().allow(null),
     testTypeEndTimestamp: Joi.date().iso().required().allow(null),
-    testExpiryDate: Joi.date().iso().allow(null),
```

Comment:
Was testExpiryDate supposed to go?

Reply:
It didn't go, I moved it to the common schema since now all the schemas should have the testExpiryDate.